### ISSUE TITLE
Invisible Dialogs when running a Toplevel with UseSystemConsole=true

### ISSUE THREAD

**Issue by tznind** [migueldeicaza/gui.cs#1698]
**Describe the bug**
When running a Toplevel only with no window showing and launch a Dialog (e.g. open file dialog) the UI is invisible when running in UseSystemConsole mode.

**To Reproduce**
```csharp
using System.Collections.ObjectModel;
using Terminal.Gui;

namespace Tests
{
    class Program
    {
        static void Main(string[] args)
        {
            Application.UseSystemConsole = (args.Length == 1 && string.Equals(args[0],"--usc"));
            Application.Init();

            var t = new Toplevel();

            Task.Run(()=>
            {
                Task.Delay(500).Wait();
                Application.MainLoop.Invoke(()=>{
                    
                    var ofd = new OpenDialog("Open", $"Select file");
                    
                    Application.Run(ofd);

                });
            });
            
            Application.Run(t);
            Application.Shutdown();
        }

    }
}
```

**Screenshots**
![invisible](https://user-images.githubusercontent.com/31306100/167247066-e75b4fcd-7d97-48ca-ba93-7d50430714db.gif)

**Desktop (please complete the following information):**
 - OS:  Linux

**Comment by BDisp** [comment]
@tznind this is already fixed by the #1685.

**Comment by tznind** [comment]
> @tznind this is already fixed by the #1685.

thats great, I will close then.  Thanks.
